.travis.yml

```diff
@@ -52,10 +52,10 @@ matrix:
           - python-gtk2
     - dist: trusty
       env:
-        - JOB="3.6, lint, coverage" ENV_FILE="ci/travis-36.yaml" TEST_ARGS="--skip-slow --skip-network" PANDAS_TESTING_MODE="deprecate" COVERAGE=true LINT=true
+        - JOB="3.6, lint, coverage" ENV_FILE="ci/travis-36.yaml" TEST_ARGS="--skip-slow --skip-network" PANDAS_TESTING_MODE="deprecate" COVERAGE=true LINT="lint|patterns"
```

Comment:
I don't think this is working as expected. `LINT` is only checked to be set, whatever is the value, would be considered as `true`. The changes in `code_checks.py` act on the `CHECK` environment variable, which is the parameter that the script receives (and different from this `LINT`.

Now the whole lint is being executed in both jobs.

I'd personally wait until #22854. We can see after the work there whether something like this still makes sense.